### PR TITLE
Update cxf bundle version

### PR DESCRIPTION
**Pull request by SujanSanjula96**
$Subject

Relates to https://github.com/wso2/product-is/issues/14289

### CODE REVIEW

File: components/org.wso2.carbon.identity.sts.passive/pom.xml [hunk 1]
```diff
@@ -96,12 +96,12 @@
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-core</artifactId>
-            <version>3.3.6</version>
+            <version>${cxf-bundle.version}</version>
```
Comment (piraveena) [inline components/org.wso2.carbon.identity.sts.passive/pom.xml:99]: remove the version from the child pom

Comment (SujanSanjula96) [reply components/org.wso2.carbon.identity.sts.passive/pom.xml]: Remove the versions from the child pom and added the dependencies to parent pom

File: pom.xml [hunk 2]
```diff
@@ -277,6 +277,48 @@
                 <artifactId>cxf-core</artifactId>
                 <version>${cxf-bundle.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.cxf.services.sts</groupId>
+                <artifactId>cxf-services-sts-core</artifactId>
+                <version>${cxf-bundle.version}</version>
+                <exclusions>
+                    <exclusion>
```
Comment (piraveena) [inline pom.xml:285]: we don't need to add these exclusions in the paren pom. if child pom are properly excluding them, then it should be fine

Comment (SujanSanjula96) [reply pom.xml]: Removed the exclusions from the parent pom

**Review by piraveena — APPROVED** [review]

**Review by piraveena — APPROVED** [review]

**Pull request merged**